.devcontainer/Dockerfile

```
@@ -0,0 +1,47 @@
+# This image version needs to match what is in the root Dockerfile.
+FROM livebook/livebook:edge
```

Since we install the dependencies in this image anyway, I don't think we need this?

```suggestion
FROM hexpm/elixir:1.13.2-erlang-24.1.7-debian-bullseye-20210902-slim
```

It's actually quite relevant, because `livebook/livebook:edge` will change with every change to livebook main, which would make the codespace image to rebuilt :D

---

Good point! I'll commit your suggestion.